AudreyW_Lab_1.ipynb

```diff
@@ -0,0 +1,598 @@
+      "text/plain": [
+       "<Figure size 360x270 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def gauss_cdf():\n",
+    "    fig, ax = plt.subplots(figsize=(5, 3.75))\n",
+    "    x = np.linspace(mu - 5*sigma, mu + 5*sigma, 1000)\n",
+    "    y = gaussCDF(x)\n",
+    "    plt.plot(x, y, ls='-', c='red', label=r'$\\mu=%i,\\ \\sigma=%i$' % (mu, sigma))\n",
+    "    plt.ylim(0, 1)\n",
+    "    plt.xlabel('$x$')\n",
+    "    plt.ylabel(r'$p(x|\\mu,\\sigma)$')\n",
+    "    plt.title('Cumulative Distribution Function for a Normal Distribution')\n",
+    "    plt.legend()\n",
+    "    plt.show()\n",
+    "gauss_cdf()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.9640696808870742\n",
+      "0.9031995154143897\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Part B, CDF Test\n",
+    "sigma_test1 = stats.norm.cdf(1.8)\n",
+    "print(sigma_test1)\n",
+    "\n",
+    "sigma_test2 = stats.norm.cdf(1.3)\n",
+    "print(sigma_test2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Values match the z-table from wikipedia"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.8000000000000003\n",
+      "-1.3000000000000003\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Part C, PPF Test\n",
+    "prob_test1 = stats.norm.ppf(1-sigma_test1)\n",
+    "print(prob_test1)\n",
+    "\n",
+    "prob_test2 = stats.norm.ppf(1-sigma_test2)\n",
+    "print(prob_test2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Got the desired sigma values"
```

Comment:
Since you got negative sigma values, it might be useful to give a brief explanation about why that is the case.